content/en/docs/about/governance.md

```diff
@@ -0,0 +1,42 @@
++++
+title =  "Governance"
+description = "About the Kubeflow Governance Structure"
+weight = 10
+aliases = ["/docs/governance/"]
++++
+
+
+## Governance Structure
+
+Kubeflow Governance consists of four components.
+
+## 1. CNCF 
+Kubeflow is a part of the Cloud Native Computing Foundation and adheres to their governance structure, which can be [found here](https://github.com/cncf/foundation?tab=readme-ov-file#governance).
+
+# 2. Kubeflow Steering Committee (KSC)
+
+At a high level, the [Kubeflow Steering Committee (KSC)](https://github.com/kubeflow/community/blob/master/KUBEFLOW-STEERING-COMMITTEE.md) is the "root" level governance body.
+
+The KSC is responsible for the general health of the Kubeflow community. Responsibilities include maintaining the governance framework, code of conduct, community meetings, communication platforms, and test resources. Read more about their [charter here](https://github.com/kubeflow/community/blob/master/KUBEFLOW-STEERING-COMMITTEE.md#charter).
```

Comment:
This seems different from what's stated in the charter?

Reply:
I pulled it from here: https://github.com/kubeflow/community/blob/master/proposals/585-kubeflow-governance/GOVERNANCE.md?plain=1

I can pull directly from the steering committee summary. 👍

Reply:
updated